### ISSUE TITLE
GG Max - Bug ?? - MIssing AKM in Object Library ??

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#3047]
Is the AKM suppose to be in the object library ? Or is this something that has been changed / removed, and this AKM selection needs removing ?

When trying to add the object to a scene, it does not add anything ?

![AKM](https://user-images.githubusercontent.com/66874403/166943761-e77dac84-4c95-4e45-b1d6-8c9771a7e37f.png)

**Comment by MonkeyFrogStudio** [comment]
Hmmm ... it's in my library.

**Comment by synchromesh62** [comment]
Same for me its gone, Renamed i think ?
That could be  the old backdrop

**Comment by ZakJudges** [comment]
@LeeBamberTGC This was due to the AKM being renamed to AK. "AKM.fpe" still needs removed from the build :)

**Comment by LeeBamberTGC** [comment]
@ZakJudges You are right, now removed all duplicate and out of date weapons from the Showcase folder for the next build
